config/config.go

```diff
@@ -716,6 +716,14 @@ type Config struct {
 	// Allows you to use custom domains
 	EnableCustomDomains bool `json:"enable_custom_domains"`
 
+	// EnableStrictRoutes changes the routing to avoid nearest-neighbour requests on overlapping routes
+	//
+	// - if disabled, `/apple` will route to `/app`, the current default behavior,
+	// - if enabled, `/app` only responds to `/app`, `/app/` and `/app/*` but not `/apple`
+	//
+	// Regular expressions and parameterized routes will be left alone regardless of this setting.
+	EnableStrictRoutes bool `json:"enable_strict_routes"`
```

Comment:
Suggest moving this flag to `HttpServerOptionsConfig` section, because we already have related flags there, for example: `SkipURLCleaning`, `SkipTargetPathEscaping` etc.

Reply:
Done! :+1: